backend/src/routes/Students.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+const studentController = require('../controllers/Students')
+const { verifyApiKey } = require('../authentication/verifyToken')
+
+const router = express.Router()
+
+// verifyToken is a middleware that checks if the user is authenticated and has the correct api-key
+router.get('/:studentId', verifyApiKey, studentController.getStudent) // get a student by id
```

Comment:
add route for getAll after adding it in controller

Reply:
done